models/webhook/webhook_test.go

```diff
@@ -72,7 +72,7 @@ func TestWebhook_EventsArray(t *testing.T) {
 		"pull_request", "pull_request_assign", "pull_request_label", "pull_request_milestone",
 		"pull_request_comment", "pull_request_review_approved", "pull_request_review_rejected",
 		"pull_request_review_comment", "pull_request_sync", "wiki", "repository", "release",
-		"package",
+		"package", "pull_review_request",
```

Comment:
To keep consistent, maybe `pull_request_review_request` is better?

Reply:
I have updated the event name to "pull_request_review_request". However, the relevant methods in the Notifier interface are named "NotifyPullReviewRequest".

Reply:
hi @lunny , plz take a look.

Reply:
> I have updated the event name to "pull_request_review_request". However, the relevant methods in the Notifier interface are named "NotifyPullReviewRequest".

Should I also change the names of the originally defined interfaces?

Reply:
I think we can do that in this PR or another PR. Both are accepted by me.

Reply:
> I think we can do that in this PR or another PR. Both are accepted by me.

In my opinion, it would be more desirable for another PR to implement this.